snuba/cli/query_replayer.py

```diff
@@ -214,19 +217,20 @@ def rerun_queries_for_blob(blob: str) -> Tuple[int, int]:
             )
 
         # File format is the same except for the directory
+        result_file_format = FileFormat(
+            directory=results_directory,
+            date=queries_file_format.date,
+            table=queries_file_format.table,
+            hour=queries_file_format.hour,
+        )
         file_manager.save(
-            FileFormat(
-                directory=results_directory,
-                date=queries_file_format.date,
-                table=queries_file_format.table,
-                hour=queries_file_format.hour,
-            ),
+            result_file_format,
             replay_results,
         )
 
         if notify:
             # TODO: maybe use new specific channel id
-            filename = file_manager.filename_from_blob_name(blob_name)
```

Comment:
this would have been getting the blob name of the queries to re-run, which is not what we want. We want the result data from the replayed queries